stockflux-securities-master/src/App.js

```diff
@@ -8,11 +8,22 @@ import { Redirect, Route, Switch } from "react-router-dom";
 import "./App.css";
 
 const App = () => {
+
+  //hard coded prop to be replaced with securities object to be edited
+  const props = {
```

Comment:
As discussed, we shouldn't need to pass any props through to the `Route`. The `securityId` should be set as a path param placeholder and dynamically provided for each row in the securities table

Reply:
yeah, changed now